examples/platform/linux/CommissionableInit.cpp

```diff
@@ -99,6 +100,35 @@ CHIP_ERROR InitConfigurationManager(ConfigurationManagerImpl & configManager, Li
         configManager.StoreProductId(options.payload.productID);
     }
 
+    if (options.vendorName.HasValue())
+    {
+        chip::Span<const char> vendor_name(options.vendorName.Value().c_str(), options.vendorName.Value().size());
+        VerifyOrDie(configManager.StoreVendorName(vendor_name) == CHIP_NO_ERROR);
```

Comment:
With these verify or die calls, is the user going to be able to tell what went wrong? This can happen fairly easily, right? If the user enters a too-long name?

Reply:
verify or die, throws error if the user does not provide value to cmd line arg. if the user enters too long name, it doesn't throw error, but when we read from the basic information cluster, we get  IM Error 0x00000501: General error: 0x01 (FAILURE). BTW its same API used for other cmd line args.